content/kb/selinux-and-longhorn.md

```diff
@@ -0,0 +1,204 @@
+---
+title: "SELinux and Longhorn"
+author: Eric Weber
+draft: false
+date: 2023-11-21
+categories:
+- "instruction"
+- "selinux"
+- "rke2"
+- "k3s"
+---
+
+## Applicable versions
+
+All Longhorn versions.
+
+## Purpose
+
+The purpose of this article is to help Longhorn users understand how Longhorn typically acts in SELinux-enabled systems
+and provide them with basic commands to run to verify normal operation. It is a work in progress that may be expanded as
+Longhorn maintainers discover additional SELinux behaviors that are of interest. It is not intended to be a guide to
+SELinux or using SELinux with Kubernetes.
+
+## SELinux basics
+
+Security-Enhanced Linux (SELinux) is a security architecture for Linux systems that allows administrators to have more
+control over who can access the system. It goes beyond discretionary access control and allows for much more granular
+decisions to be made about which processes can access files or communicate with other processes. Many distributions
+support SELinux, and some even enable it by default. Consult the documentation for your distribution to determine
+whether SELinux is supported and how to enable it.
+
+When SELinux is enabled, it uses policies to determine whether or not accesses are allowed. Most distributions ship
+default policies, but administrators are free to write their own or modify the pre-existing ones. In addition, some
+software ships with SELinux policies that grant permissions necessary for operation.
+
+## SELinux and Kubernetes
+
+Many popular container runtimes and Kubernetes distributions support SELinux. The upstream
+[container-selinux](https://github.com/containers/container-selinux) project defines the SELinux
+[domains/types](https://danwalsh.livejournal.com/81756.html) that container processes and the files they interact with
```

Comment:
Perhaps "... [domains/types] for container processes and the files they interact with."

Reply:
Good call. Fixed this and squashed everything.